src/shared/AuthButtons/AuthButtons.js

```diff
@@ -1,6 +1,7 @@
-import React, { useRef, useEffect, useCallback } from "react";
+import React, { useRef, useEffect, useCallback, useState } from "react";
 import styled from "styled-components";
 import { ActionButton } from "../Buttons";
+import { ErrorPopup } from "../../components/ErrorPopup.js";
```

Comment:
Also you may consider import ErrorPopup directly since it is exported with the default keyword.